modules/database/src/ioc/dbStatic/dbStaticLib.c

```diff
@@ -743,8 +743,8 @@ static long dbAddOnePath (DBBASE *pdbbase, const char *path, unsigned length)
     ppathList = (ELLLIST *)pdbbase->pathPvt;
 
     pdbPathNode = (dbPathNode *)dbCalloc(1, sizeof(dbPathNode));
-    pdbPathNode->directory = (char *)dbCalloc(length+1, sizeof(char));
-    strncpy(pdbPathNode->directory, path, length);
+    pdbPathNode->directory = (char *)dbMalloc(length+1);
+    strncpy(pdbPathNode->directory, path, length+1);
```

Comment:
Since the length is already known exactly, this could become a `memcpy()`, which would also silence this warning.